janusgraph-core/src/main/java/org/janusgraph/graphdb/database/serialize/StandardSerializer.java

```diff
@@ -17,6 +17,8 @@
 import com.google.common.base.Preconditions;
 import com.google.common.collect.BiMap;
 import com.google.common.collect.HashBiMap;
+import org.apache.tinkerpop.shaded.jackson.databind.node.ArrayNode;
```

Comment:
Please move these imports down with the other Tinkerpop imports, preferably alphabetized.

Reply:
@chupman Thank you for the review! Moved those imports down with the other Tinkerpop imports. Alphabetized by the full class path.